docs/react/ssr.md

```
@@ -8,4 +8,68 @@ slug: /react/ssr
 
 # Server-Side Rendering  
 
-Currently project is not support SSR.  
\ No newline at end of file
+SSR is supported with **ViewModelStore** and predictable hydration on the client.  
+The main requirement is to keep the initial render **identical** on server and client.  
+
+#### Next.js  
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)